src/components/atoms/UiCheckbox/UiCheckbox.vue

```diff
@@ -172,7 +172,7 @@ const checkboxId = computed(() => (props.id || `checkbox-${uid()}`));
 const isChecked = computed(() => {
   if (Array.isArray(props.modelValue)) {
     return !!props.modelValue.find((option) => (
-      equal(props.value, option)));
+      equal(JSON.parse(JSON.stringify(props.value)), JSON.parse(JSON.stringify(option)))));
```

Comment:
If the problem is on top layer, `Object.assign({}, target)` or `{ ...target }` are significantly more performant at "plain"ing the object.

Reply:
Nope, the problem is deeper. In for example 'suffixAttrsProps' and `onClick` function